5] dice/main.py

```diff
@@ -0,0 +1,36 @@
+import re
+from random import randint
+
+def roll(kostky):
```

Comment:
ufff.. no jako.. celkem komplikovane zase, ale ocenuju ze je videt zes to asi delala sama a nejak o tom premyslela. 
Rozhodne to jde zjednodusit, kdyztak se podivej jak, ja jsem nejak posilal svoji vetev.